src/components/Button.js

```diff
@@ -13,8 +13,9 @@ const StyledButton = styled.button(
     display: inline-flex;
     align-items: center;
     justify-content: center;
-    user-select: none;
-    height: ${BUTTON_HEIGHT}px;
+    box-sizing: border-box;
```

Comment:
Do we need border-box here? Isn't these covered by the global reset?

I would recommend adding "user-select: none" some touch screen devices make it easy to select the button text when the user is trying to press the button. Unless there's a good use case for a user wanting to select the button text?

Reply:
> Do we need border-box here? Isn't these covered by the global reset?

Yeah unfortunately the `all: unset` rule undoes it. On my personal projects I've gone off this rule and instead just remove `border` and `background` etc.

> I would recommend adding "user-select: none"

Oops, wasn't meant to remove :sweat_smile: I had crazy merge conflicts going on all over the place :grimacing: